pvlib/scaling.py

```diff
@@ -134,6 +134,7 @@ def fn(x):
         return np.abs((x ** 2 - x) / 2 - n_pairs)
 
     n_dist = np.round(scipy.optimize.fmin(fn, np.sqrt(n_pairs), disp=False))
+    n_dist = n_dist.item()
```

Comment:
This change needs to be checked by someone who understands the WVM code better than I do.  @cwhanse?

Reply:
Line 136 returns an array of length 1, the `item` is converting to scalar. Is that correct? If so, then this looks approriate to me.

I don't think we would have triggered the error described in the numpy warning at line 145/6, since `n_dist.ndim` woudl be 1. Better to be specific here, in any case.